koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -619,7 +619,11 @@ class BuildContainerTask(BaseContainerTask):
                     "triggered_after_koji_task": {
                         "type": "integer",
                         "description": "Koji task for which autorebuild runs"
-                    }
+                    },
+                    "userdata": {
+                        "type": ["object"],
```

Comment:
Slightly better is `"type": "object",`

Reply:
I was matching the previous usage but sure.